<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Particulas</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLineEdit" name="id_lineEdit"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Origen en Y</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="y1_spinBox_3"/>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Origen en  X</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QSpinBox" name="x1_spinBox_2"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Destino en Y</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="y2_spinBox_4"/>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Destino en  X</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QSpinBox" name="x2_spinBox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Velocidad</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QSpinBox" name="velocidad_spinBox_5"/>
           </item>
           <item row="3" column="4" rowspan="5" colspan="3">
            <widget class="QPlainTextEdit" name="salida_plainTextEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Rojo</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QDoubleSpinBox" name="rojo_doubleSpinBox"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Azul</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QDoubleSpinBox" name="azul_doubleSpinBox_2"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_10">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Verde</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QDoubleSpinBox" name="verde_doubleSpinBox_3">
             <property name="value">
              <double>30.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QPushButton" name="AgreIni_pushButton">
             <property name="text">
              <string>Agregar al Inicio</string>
             </property>
            </widget>
           </item>
           <item row="8" column="4">
            <widget class="QPushButton" name="AgreFin_pushButton_2">
             <property name="text">
              <string>Agregar al Final</string>
             </property>
            </widget>
           </item>
           <item row="8" column="6">
            <widget class="QPushButton" name="Mostrar_pushButton_3">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="tablaP"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="BuscarT_lineEdit">
          <property name="text">
           <string>Buscar particula...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="BuscarP_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="MostrarP_pushButton_2">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
